www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/forms/naxsi_rule.xml

```diff
@@ -43,68 +46,75 @@
     </field>
     <field>
         <id>naxsi_rule.args</id>
-        <label>Arguments</label>
+        <label>Search in any GET Argument</label>
```

Comment:
Do we really need such a long string or is "GET Argument" enough?

Reply:
When I first used the plugin I was confused by a yes/no checkbox which states "Arguments". _Does this mean "Arguments are present"?_ was my first thought. To improve this I tried to make the labels more self-explanatory.